pkg/unikontainers/unikontainers.go

```diff
@@ -528,6 +540,7 @@ func (u *Unikontainer) ExecuteHooksSequentially(name string) error {
 		"Poststart":       u.Spec.Hooks.Poststart,
 		"Poststop":        u.Spec.Hooks.Poststop,
 	}[name]
+	Log.WithField("hooks", name).WithField("amount", len(hooks)).Error("")
```

Comment:
Let's not use Error logs for non-error cases. We can decrease the log level to Info or debug.

Reply:
Sure, I changed the log level.